tsconfig.json

```diff
@@ -12,9 +12,5 @@
     "target": "es5",
     "noEmitOnError": true,
     "noImplicitAny": true,
-  },
-  "include": [
-    "./index.ts",
-    "./src/**/*"
-  ]
```

Comment:
just curious, why's this not needed anymore?

Reply:
I read on the internets if you don't include "include" it just includes everything by default.